openshift_cli_installer/manifests/clusters.example.yaml

```diff
@@ -144,3 +144,23 @@ clusters:
   worker-flavor: m5.4xlarge
   worker-root-disk-size: 128
   log_level: info # optional, default: "error", supported options are debug, info, warn, error
+
+# ARO cluster
+- name: msi-aro-c1
+  platform: aro
+  version: "4.13.23"
+  region: eastus
+  stream: stable
+  timeout: 1h
+  domain: msi # Domain name must contain 1 to 30 alphanumeric characters or '.', and start and end with an alphabetic character
```

Comment:
the comment is about valid domains; please remove.
add to readme that domain must be a valid one

Reply:
done